renderer/src/common/components/workloads/dialog-workload-form-wrapper.tsx

```diff
@@ -52,11 +54,15 @@ export function DialogWorkloadFormWrapper<T extends FieldValues = FieldValues>({
             <form onSubmit={onSubmit} className="flex min-h-0 flex-1 flex-col">
               <DialogHeader className="flex-shrink-0 p-6 pb-4">
                 <DialogTitle>{title}</DialogTitle>
-                <DialogDescription className="sr-only">
-                  ToolHive allows you to securely run a remote MCP server or a
-                  custom local MCP server from a Docker image or a package
-                  manager.
-                </DialogDescription>
+                {description ? (
+                  <DialogDescription>{description}</DialogDescription>
+                ) : (
+                  <DialogDescription className="sr-only">
```

Comment:
i don't know if we even need to keep this screen reader only text here? 🤔 does this actually provide any real benefit to screenreader users? it does not seem immediately obvious how 🤔